src/main/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/ConjugatedPiSystemFragmenter.java

```diff
@@ -0,0 +1,256 @@
+package de.unijena.cheminf.mortar.model.fragmentation.algorithm;
+
+
+import de.unijena.cheminf.mortar.gui.util.GuiUtil;
+import de.unijena.cheminf.mortar.message.Message;
+import de.unijena.cheminf.mortar.model.util.SimpleEnumConstantNameProperty;
+import javafx.beans.property.Property;
+import javafx.beans.property.SimpleBooleanProperty;
+import org.openscience.cdk.exception.CDKException;
+import org.openscience.cdk.graph.invariant.ConjugatedPiSystemsDetector;
+import org.openscience.cdk.interfaces.IAtomContainer;
+import org.openscience.cdk.interfaces.IAtomContainerSet;
+import org.openscience.cdk.tools.CDKHydrogenAdder;
+
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Java class implementing an algorithm in MORTAR for detection of conjugated pi systems using
+ * a rarely used CDK functionality to test and validate its purpose.
+ *
+ * @author Maximilian Rottmann
+ */
+public class ConjugatedPiSystemFragmenter implements IMoleculeFragmenter{
```

Comment:
Ok, I'd say we have to discuss whether we really want to keep this fragmenter in MORTAR. If no, we have to throw it away. If yes, you probably know that you still have to "make it pretty". I will not review it now, therefore. But you can already transfer a lot of the comments on the other fragmenter to this one.